app/poros/github_contributors.rb

```diff
@@ -0,0 +1,11 @@
+class GithubContributors
+
+  def self.contributors_info
+    github_contributor_data = GithubService.contributors_info
+    github_contributor_data.each_with_object({}) do |contributor, hash|
+      if [26797256, 78388882, 5446926, 78574189].include?(contributor[:id])
+        hash[contributor[:id]] = {name: contributor[:name], contributions: contributor[:contributions]}
+      end
+    end
+  end
+end
```

Comment:
Yay! I think this looks really nice. Proud of us as a team.

Reply:
Together we are strong!